image_tools/src/showimage.cpp

```diff
@@ -17,22 +17,28 @@
 #include <sstream>
 #include <string>
 
-#include "opencv2/highgui/highgui.hpp"
-#include "opencv2/imgproc/imgproc.hpp"
-
 #include "rclcpp/rclcpp.hpp"
 
 #include "sensor_msgs/msg/image.hpp"
 
 #include "image_tools/options.hpp"
 
+#include "image_tools/showimage.hpp"
+
+#include "opencv2/highgui/highgui.hpp"
```

Comment:
style nit: move opencv header includes where they were (above rclcpp/rclcpp.hpp). See [style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) for order of includes.